Classes/person.rb

```diff
@@ -30,4 +31,8 @@ def can_use_services?
   def validate_name
     @corrector.correct_name
   end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] `The person` itself should be sent as a parameter to create the new rental achieving the 'has-many' association. New rental should be created with `date`, `book`, and `person` itself

Reply:
Do you mean writing  :
rentals.person = self